.github/workflows/codeql.yml

```
@@ -0,0 +1,57 @@
+name: CodeQL
+
+on:
+  push:
+    branches:
+      - main
+      - release/v*
+      - feat/ci/codeql-and-scorecard
+  pull_request:
+    branches:
+      - main
+      - release/v*
+      - feat/ci/codeql-and-scorecard
```

```suggestion
      - "release/**"
  pull_request:
    branches:
      - main
      - "release/**"
```
Why shouldn't this always run on PR?